Packet++/header/SSLLayer.h

```diff
@@ -129,30 +130,34 @@
  *
  * __Cipher suites:__    <BR>
  *
- * Cipher suites are named combinations of authentication, encryption, message authentication code (MAC) and key exchange
- * algorithms used to negotiate the security settings for a network connection using SSL/TLS.
+ * Cipher suites are named combinations of authentication, encryption, message authentication code (MAC) and key
+ * exchange algorithms used to negotiate the security settings for a network connection using SSL/TLS.
  * There are many known cipher-suites. PcapPlusPlus support above 300 of them, according to this list:
  * http://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml
- * There is a designated class in PcapPlusPlus called pcpp::SSLCipherSuite which represents the cipher-suites and provides
- * access to their attributes. Then there is a static instance of this class for each one of the supported cipher-suites.
- * This means there are 300+ static instances of pcpp::SSLCipherSuite representing the different cipher suites. The user can
- * access them through static methods in pcpp::SSLCipherSuite or from client-hello and server-hello messages where they appear
+ * There is a designated class in PcapPlusPlus called pcpp::SSLCipherSuite which represents the cipher-suites and
+ * provides
```

Comment:
not sure why clang-format added a line break after the word `provides` 🤔

Reply:
The line was too long. manually modified.